site/blog/starlingx-release-2-security-features.md

```diff
@@ -0,0 +1,23 @@
+---
+title:  Securing the Edge with StarlingX
+author: ian-jolliffe
+date: 2019-10-30T01:32:05.627Z
+category: features-and-updates
+---
+Security on the edge is crucial. Learn more about how the StarlingX 2.0 release helps addressing this challenge. <!-- more -->
+
+
+As we drive compute to the edge of the network many things change and become more interesting and new problems need to be solved. At a minimum the challenges include space and power limitations which then turns into balancing server costs while enabling emerging applications and new transformational use cases at the edge when you take the next steps. However; even though the way compute is delivered at the edge is different, the need for security remains constant or even more critical.
+
+Security is always an important consideration in any deployment. It gets more interesting when we no longer have the high level of physical security in the data center where there are retinal scanners at the doors. As we move to the edge we have an environment where there may be less physical security and nodes that are geographically distributed also make maintenance activities difficult.
+
+Within the StarlingX community these are challenges we have recognized and have been working on for a while to provide solutions for. The key approaches start with security enabled at the lowest level of the stack and build up from there. The key challenges we have tackled so far are:
+
+- **UEFI secure boot**: Can be used to securely boot from the lowest levels of the stack such that the operating system can be booted from BIOS to GRUB in a secure way.
+- **TPM device enablement**: StarlingX provides the ability to leverage a TPM device when it is available on a server. TPM devices can be used to store secrets, such as private keys for HTTPS. Once they are in the TPM device, the secret can only be read by authorized users and can never be modified. Another important use of TPM devices is to leverage entropy built into the device to ensure more randomness of any generated keys. 
+- **Certificate management**: In a Kubernetes environment the number of certificates can easily explode. We have some initial capabilities with StarlingX Release 2.0 where these certificates are managed more manually than we would like. In future releases, we are looking to collaborate with other projects to make it easier to manage these. 
```

Comment:
In future releases, we are looking to collaborate with other projects to make it easier to manage these certificates.